### ISSUE TITLE
we should switch from sipdistutils to a project.toml file

### ISSUE THREAD

**Issue by wbsoft** [frescobaldi/python-poppler-qt5#38]
As per https://www.riverbankcomputing.com/static/Docs/sip/introduction.html we should go with a project toml file, instead of the sipdistutils hack hack. Currently investigating. Installing with pip should work.

**Comment by tadeoos** [comment]
Thanks for all the work on this package! Any idea when this could happen?
pip installation, unfortunately, does not work and the way around it is to install via apt-get: `apt-get install -y python3-poppler-qt5`, but this, in turn, does not support python3.8

**Comment by mitya57** [comment]
I started some work on this in the sip5 branch of my fork:
https://github.com/mitya57/python-poppler-qt5/commits/sip5

So far it's very raw, so I don't even create a draft pull request.

What is missing:
- Does not use pkg-config, has hardcoded include path.
- `version.sip` is hard-coded, not generated.
- Does not set POPPLER_VX_Y_Z version tag.

I will try to finish when I have more time, or feel free to start from where I stopped.
